(StorybookUtils): LiveCodeExample - set overflow to visible

mosheKerbel opened the pull request:
@argshook @shlomitc 

before:
![Screen Shot 2020-10-26 at 9 19 57](https://user-images.githubusercontent.com/7388217/97144123-9219c000-176c-11eb-88b1-dddaa6fced8e.png)


after:
![Screen Shot 2020-10-26 at 9 19 41](https://user-images.githubusercontent.com/7388217/97144128-947c1a00-176c-11eb-83f3-071fda42b69b.png)

shlomitc commented:
The reason for the overflow is to make the full-screen playground (like in WSR) to support scrolling.
In the inline code-snippets, bad things happen.

In WSR I just removed all these dashed code examples in favor of the plain background ones (also updated the generator).
Anyway, I think it's better to have only a single type of inline code examples.

argshook commented:
> The reason for the overflow is to make the full-screen playground (like in WSR) to support scrolling.
> In the inline code-snippets, bad things happen.

maybe `overflow: auto` should be applied only for playground, and all
the rest of code examples don't specify overflow at all (thus, the
default `overflow: visible` would work)?

mosheKerbel commented:
> The reason for the overflow is to make the full-screen playground (like in WSR) to support scrolling.
> In the inline code-snippets, bad things happen.
> 
> In WSR I just removed all these dashed code examples in favor of the plain background ones (also updated the generator).
> Anyway, I think it's better to have only a single type of inline code examples.

When we started documenting base-ui, @Zohar-Telor requested the bg-color to be transparent for components, so it's working that way for all pages in base-ui.

myabe we can do what Arijus suggested?

shlomitc commented:
yes, I'm good with this solution as well.

argshook commented on packages/wix-storybook-utils/src/LiveCodeExample/LiveCodeExample.tsx:
```diff
@@ -37,6 +37,7 @@ export interface Props {
   autoRender?: boolean;
   darkBackground?: boolean;
   noBackground?: boolean;
+  isOverflow?: boolean;
```
@mosheKerbel i don't think you need a dedicated prop, just remove `overflow` css property entirely and add it to playground only

argshook reviewed (approved):
:+1:
